src/views/blocks/tickets/quantity-add.php

```diff
@@ -13,20 +13,21 @@
  * @since 4.9.3
  * @since 4.11.3 Updated the button to include a type - helps avoid submitting forms unintentionally.
  * @since 4.11.4 Added accessibility classes to screen reader text element.
+ * @since TBD    Removed duplicate `type="button"` from button element.
  *
- * @version 4.11.4
+ * @version TBD
  *
  * @var $this Tribe__Tickets__Editor__Template
  */
 
 $ticket = $this->get( 'ticket' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space

<sub>hash: 454b45d1160d913234476ad142b23c7b</sub>